xml/System/Environment.xml

```diff
@@ -1753,13 +1753,20 @@ To identify the operating system platform, for example, Linux or Windows, you ca
         <ReturnType>System.Int32</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>Gets the number of processors on the current machine.</summary>
-        <value>The 32-bit signed integer that specifies the number of processors on the current machine. There is no default. If the current machine contains multiple processor groups, this property returns the number of logical processors that are available for use by the common language runtime (CLR).</value>
+        <summary>Gets the number of processors available to the current process.</summary>
+        <value>The 32-bit signed integer that specifies the number of processors that are available.</value>
         <remarks>
           <format type="text/markdown"><![CDATA[
 
 ## Remarks
- For more information about processor groups and logical processors, see [Processor Groups](/windows/win32/procthread/processor-groups).
+On Linux and macOS systems for all .NET versions and on Windows systems starting with .NET 6, this API returns the minimum of:
+- The physical number of CPUs on the machine 
```

Comment:
> - The physical number of CPUs

I am still not sure what "the physical number of CPUs" means. If it means "the number of physical CPUs", then it is not the number we actually use in the code.  The [documentation](https://docs.microsoft.com/en-us/windows/win32/api/sysinfoapi/ns-sysinfoapi-system_info) for `SYSTEM_INFO`'s `dwNumberOfProcessors` uses the term "logical processors" and distinguish them from "physical processors":

> The number of **logical processors** in the current group. To retrieve this value, use the GetLogicalProcessorInformation function.
>
> Note  For information about the **physical processors** shared by logical processors, call GetLogicalProcessorInformationEx with the RelationshipType parameter set to RelationProcessorPackage (3).

On my machine Task Manager displays 4 cores (physical processors) and 8 logical processors:
```
Sockets:              1
Cores:                4
Logical processors:   8
```

Reply:
Ah. I misunderstood what you meant, earlier. There was a spelling mistake as well. That's what I thought you were referring to.

```suggestion
- The logical number of processors on the machine 
```

Reply:
I think it should be "logical processor", not "logical number".

Reply:
How about `The number of logical processors on the machine` ??

Reply:
I see you proposed the same thing. I committed it.